### DISCUSSION TITLE
Audibookshelf as a Music Provider possible?

### DISCUSSION THREAD

**Discussion by der10mrx** [music-assistant/support#2200] [Feature requests and ideas]
Hello!
I was wondering if you can integrate Audiobookshelf https://github.com/advplyr/audiobookshelf as a provider even though it is not a music provider. 
I'd love to use Music Assistant to listen to my audiobooks.

**Comment by OzGav** [comment]
Please search for related discussions first https://github.com/orgs/music-assistant/discussions/429

**Reply by jnbht** [reply]
That is not the same feature request.
Audiobookshelf is a Self-hosted audiobooks server!

**Reply by OzGav** [reply]
I know that it is but the feature to support any sort of audiobook needs to be added first. So you should be upvoting that post and probably could have just added this to there but no matter

**Comment by igbjnI05bF** [comment]
I would absolutely love if AudioBookShelf was available as a source. Preferably added in a way that keeps track of listening positions, so that I could be listening to an audiobook or podcast in my car, then walk inside and resume playing on my home speakers via musicassistant/snapcast.
